pokemon-app/src/components/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+import Pokemons from './Pokemons'
+import { getPokemonsLength } from '../../selectors'
+
+import '../../styles/gallery.scss'
+
+const Gallery: React.FC = () => {
+    const pokemonsLength: number = useSelector(getPokemonsLength)
```

Comment:
Please rename as pokemonsLengthSelector & all selectors should have ...selector postfix